packages/thumbprint-tokens/src/__snapshots__/index.test.js.snap

```diff
@@ -362,3 +362,105 @@ $tp-wrap__no-pad-width: 1010px;
 $tp-z-index__modal: 200;
 "
 `;
+
+exports[`outputted iOS matches snapshot 1`] = `
+"import UIKit.UIColor
+
+/// These are the colors we use at Thumbtack.
+public enum Color {
+    /// Maps to #eaf6fa
+    public static let blue100: UIColor = UIColor(red: 0.91764706, green: 0.9647059, blue: 0.98039216, alpha: 1.0)
+    /// Maps to #b3ebff
```

Comment:
nit: Can we add an empty line between each property (e.g., in between lines 372 & 373 here)? Makes it easier to read when inspecting the source during iOS . development